rllib/env/single_agent_env_runner.py

```diff
@@ -166,7 +174,6 @@ def sample(
                 num_episodes=num_episodes,
                 explore=explore,
                 random_actions=random_actions,
-                with_render_data=with_render_data,
```

Comment:
So render data should not anymore go through the episodes but directly through a callback?

Reply:
Correct. We will not natively support rendering, only through custom callbacks and this new metrics logging system.